arch/platform/cc26x0-cc13x0/platform.c

```diff
@@ -172,7 +173,11 @@ platform_init_stage_one()
 void
 platform_init_stage_two()
 {
-  random_init(0x1234);
+  soc_trng_init();
+
+  /* Use TRNG to seed random lib. */
+  uint16_t seed = (uint16_t)soc_trng_rand_synchronous();
```

Comment:
Can this variable be inlined, or does that make the next line too long?

Reply:
I was actually intentional doing it this way to make it clearer. But now with some daylight, and the comment above, I think it is more than clear enough when inlined 👍